### PR TITLE
[WIP] Adapt test_verify_revert_snapshot for RHV

### PR DESCRIPTION
**Pull request by bargulg**
test_verify_snapshot_revert should work in RHV now...
Also, removed bugzilla markers for closed bugs

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 1b822c0bd1df0db2ede9d69dc6bd403291df2a14:

`cfme/tests/infrastructure/test_snapshot.py`:
- :bangbang: Line 98:43: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/infrastructure/virtual_machines.py [hunk 1]
```diff
@@ -132,24 +132,23 @@ def _nav_to_snapshot_mgmt(self):
                 sel.click(InfoBlock.element("Properties", "Snapshots"))
 
         def does_snapshot_exist(self):
+            name = self.name or self.description
```
Comment (mfalesni) [inline cfme/infrastructure/virtual_machines.py]: Are you sure about the precedence order in here? I know that in ManageIQ usually description is the one that takes precedence when specified.

Comment (bargulg) [inline cfme/infrastructure/virtual_machines.py]: this method needs to use name to find snapshot if name was specified, same as it does now, and description is just a fallback if name wasn't specified (like with RHV, where snapshots have description, but no name)

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fd070073773411b2ae2f44b053451ce01c135dd8:

Lint OK! :cake: :punch: :cookie: .

* fd070073773411b2ae2f44b053451ce01c135dd8
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ef81a9b27066a3128500bbb3419b156fe26fbaac:

`cfme/tests/infrastructure/test_snapshot.py`:
- :bangbang: Line 115:1: **E305** *expected 2 blank lines after class or function definition, found 0*
- :bangbang: Line 115:2: **E999** *SyntaxError: invalid syntax*
- :bangbang: Line 115:3: **E227** *missing whitespace around bitwise or shift operator*
- :bangbang: Line 115:7: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by jkrocil** [comment]
Closing; @slaykovsky feel free to open a new PR.

**Comment by slaykovsky** [comment]
@jkrocil this one is already done :)

**Pull request closed without merging**